src/js/AccessibilityUtils.js

```diff
@@ -577,6 +577,40 @@ axs.utils.hasLabel = function(element) {
 };
 
 /**
+ * Determine if this element is disabled without considering the effect of a disabled ancestor.
+ * Disabled here means that the element should be considered disabled according to specification.
+ * This element may or may not be effectively disabled in practice as this is dependent on implementation.
+ * @param {Element} element An element to check.
+ * @return {boolean} True if the element is disabled.
+ */
+axs.utils.isElementDisabled = function(element) {
+    if(!element || element.nodeType !== Node.ELEMENT_NODE) {
```

Comment:
Given that this is being explicitly called with an `Element` object, we probably don't need to check `nodeType`.